src/resilience/exec_space/openMP/OpenMPResParallel.hpp

```diff
@@ -127,41 +193,106 @@ class ParallelFor< FunctorType
       auto m_functor_1 = m_functor;
```

Comment:
I guess I didn't notice this before but a better name would be something like `functor_copy_0` and `functor_copy_1`, or even better yet an array like:

```cpp
auto functor_copies = std::array{ m_functor, m_functor };
```

Reply:
a std::array would generate identical code here so no difference in performance